File: app/system/modules/DateTimeHandlerModule.java
package system.modules;

import com.google.inject.AbstractModule;
import util.datetime.DateTimeHandler;
import util.datetime.DateTimeHandlerImpl;

public class DateTimeHandlerModule extends AbstractModule {

    @Override
    protected void configure() {
        bind(DateTimeHandler.class).to(DateTimeHandlerImpl.class);
    }
}

File: app/system/modules/DateTimeHandlerModule.scala
// SPDX-FileCopyrightText: 2024 The members of the EXAM Consortium
//
// SPDX-License-Identifier: EUPL-1.2

package system.modules

import com.google.inject.AbstractModule
import services.datetime.{DateTimeHandler, DateTimeHandlerImpl}

class DateTimeHandlerModule extends AbstractModule:
  override def configure(): Unit = bind(classOf[DateTimeHandler]).to(classOf[DateTimeHandlerImpl])
